src/ripple/app/tests/TxQ_test.cpp

```diff
@@ -1529,6 +1530,42 @@ class TxQ_test : public beast::unit_test::suite
         }
     }
 
+    void testRPC()
+    {
+        using namespace jtx;
+        {
+            Env env(*this, features(featureFeeEscalation));
+
+            auto fee = env.rpc("fee");
+
+            if (expect(fee.isMember(jss::result) &&
+                !RPC::contains_error(fee[jss::result])))
+            {
+                auto const& result = fee[jss::result];
+                expect(result.isMember(jss::drops) &&
+                    result.isMember(jss::levels) &&
+                    result.isMember(jss::current_ledger_size) &&
+                    result.isMember(jss::current_queue_size) &&
+                    result.isMember(jss::expected_ledger_size));
```

Comment:
I would break each of these out into its own call to `expect` so if one fails you'll know which